src/app/components/add-card/add-card.component.html

```diff
@@ -0,0 +1,33 @@
+<app-create-button (createButtonClick)="showInput()"></app-create-button>
+
+<div *ngIf="utilityService.show">
+  <input [(ngModel)]="newTask" #newTaskInput />
+  <button (click)="addTaskToTaskList()" id="addTaskButton">Add Task</button>
+  <button (click)="resetInput()">
+    <i class="fas fa-trash-alt" style="color: rgba(109, 107, 107, 0.835)"></i>
```

Comment:
I would avoid inline style and add class as it will make the markup more cleaner.

Reply:
done!